.github/workflows/npm-publish.yml

```diff
@@ -55,14 +55,23 @@ jobs:
               for PACKAGE in $PACKAGES
               do
                   echo "Publishing $PACKAGE..."
-                  cd $PACKAGE && npm publish && cd -
-
-                  echo "Creating GitHub release for $PACKAGE..."
-                  PACKAGE_NAME=$(basename $PACKAGE)
+                  
+                  cd $PACKAGE
+                  
                   PACKAGE_VERSION=$(node -p "require('./$PACKAGE/package.json').version")
+                  PACKAGE_NAME=$(basename $PACKAGE)
                   PACKAGE_TAG="${PACKAGE_NAME}_v${PACKAGE_VERSION}"
                   PACKAGE_BODY="TODO FILL OUT RELEASE NOTES for $PACKAGE_NAME v$PACKAGE_VERSION"
 
+                  if [[ $PACKAGE_VERSION == *"rc"* ]]
+                  then
+                    npm publish --tag rc && cd -
+                  else
+                    npm publish && cd -
```

Comment:
does this mean it _wont_ be installed with `npm install @photon-health/sdk@latest` ?

Reply:
<img width="934" alt="Screen Shot 2024-05-23 at 2 45 58 PM" src="https://github.com/Photon-Health/client/assets/700617/207b3da2-ca70-4ef4-b01d-e50725d91725">
Yup, according to this rando https://www.iliascreates.com/blog/post/npm-distribution-tags/